src/config/network/iwd.md

```diff
@@ -13,59 +13,62 @@ Install the `iwd` package and enable the `dbus` and `iwd` services.
 
 ## Usage
 
-IWD provides a command line client, `iwctl`. It can be used to add, remove, and
-configure network connections; running it produces an interactive prompt. To
-list available options, enter `help`.
+IWD provides a command line client, `iwctl(1)`. It can be used to add, remove,
+and configure network connections; running it produces an interactive prompt. It
+also supports passing commands as arguments. To list available options, enter
+`help`.
```

Comment:
Not sure about the last sentence: does it refer to typing `help` at the interactive prompt, as an argument on the command line, or both?

Suggest putting the "It also supports ..." sentence last in this para.

Reply:
It's both. I'm not sure how to phrase it either. Do you have a suggestion?

Reply:
How about:

"The command-line client \`iwctl(1)\` can be used to add, remove,
and configure network connections. Commands can be passed as arguments; when run without arguments, it provides an interactive session. To list available commands, run either \`iwctl help\`, or enter \`help\` at the interactive prompt."

Reply:
Awesome!